pkg/controller/build/buildrequest/buildrequest.go

```diff
@@ -301,15 +322,21 @@ func (br buildRequestImpl) toBuildahPod() *corev1.Pod {
 			Name:  "BUILDAH_ISOLATION",
 			Value: "chroot",
 		},
+		{
+			Name:  "HTTP_PROXY",
+			Value: httpProxy,
+		},
+		{
+			Name:  "HTTPS_PROXY",
+			Value: httpsProxy,
+		},
+		{
+			Name:  "NO_PROXY",
+			Value: noProxy,
+		},
 	}
 
-	var uid int64 = 1000
-	var gid int64 = 1000
-
-	securityContext := &corev1.SecurityContext{
-		RunAsUser:  &uid,
-		RunAsGroup: &gid,
```

Comment:
question: why are we removing the uid/gid configuration here?

Reply:
because we begin with a root user in an unprivileged container so that we can update the PEM bundles using `p11-kit extract` and then a switch to the build user happens in the buildah script to the `build` user